packages/tools/pie-wrappers-react/scripts/react-remove-wrapper.js

```diff
@@ -0,0 +1,21 @@
+import { readFile, writeFile } from 'fs';
+import { components } from './react-gen-wrapper.js';
+
+// removes react wrapper from index.ts after dist has built
+function removeReactWrapper (component) {
+    readFile(`../../components/${component.tagName}/src/index.ts`, { encoding: 'utf8', flag: 'r' }, (err, data) => {
```

Comment:
This can be simplified by dropping the callback, since we can use `readfileSync`.